release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -71,6 +71,8 @@ See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades
 phased roll out] for more information on the types of updates and upgrade
 channels.
 
+Because upgrades are published to the channels on a time delay as they are gradually rolled out to customers based on data from our SRE team, you might not immediately be able to update from version 4.1.z to 4.2 at initial release.
```

Comment:
They are "able to update", just not via the button consuming suggestions from the cloud.  We might alter the second half to say "you might not immediately see notification in the UI at initial release" to more accurately capture what's going on there.

Reply:
Though it still begs the question of when the fast channel upgrade will come, here are some thoughts: 

Red Hat recommends [or We recommend] timing your upgrade from 4.1.z to 4.2.0 to coincide with a fast channel upgrade. Fast channel upgrades are published on a time delay and gradually rolled out to customers based on data from our SRE team, which is why you might not see notification in the UI at the initial release time.

Alternately you could add a sentence warning them that if they decide to proceed with an upgrade, we strongly recommend doing a full backup?